lib/manifest.ps1

```diff
@@ -7,7 +7,15 @@ function manifest_path($app, $bucket) {
 
 function parse_json($path) {
     if(!(test-path $path)) { return $null }
-    Get-Content $path -raw -Encoding UTF8 | convertfrom-json -ea stop
+    $json = Get-Content $path -raw -Encoding UTF8 | convertfrom-json -ea stop
+    if ($json.innosetup) {
```

Comment:
I am not fan of doing this in parse function.

Why not rather move this workaround into `install_app` function??

https://github.com/lukesampson/scoop/blob/1ac8ac0c6506f8012e18ba143f315abcc95ef836/lib/install.ps1#L12

There you can just do completely same thing.

Filtering objects while parsing manifest is not good idea overall.